src/main.tsx

```diff
@@ -1,12 +1,32 @@
 import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
+import { RouterProvider, createBrowserRouter } from "react-router";
 import "./index.css";
 import "react-tooltip/dist/react-tooltip.css";
 import App from "./App.tsx";
+import { DealPreparation } from "./pages/DealPreparation";
+import { Download } from "./pages/Download";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <App />,
+    children: [
+      {
+        path: "/",
+        element: <DealPreparation />,
+      },
+      {
+        path: "/download",
+        element: <Download />,
+      },
+    ],
+  },
+]);
```

Comment:
```suggestion
const router = createBrowserRouter([
  {
    path: "/",
    Component: App,
    children: [
      {
        index: true,
        Component: DealPreparation,
      },
      {
        path: "/download",
        Component: Download,
      },
    ],
  },
]);
```

https://reactrouter.com/start/data/routing

Looks like it'd be more idiomatic way of doing that.